network/network_windows.go

```diff
@@ -427,8 +426,8 @@ func (nm *networkManager) newNetworkImplHnsV2(nwInfo *NetworkInfo, extIf *extern
 	if err != nil {
 		// if network not found, create the HNS network.
 		if errors.As(err, &hcn.NetworkNotFoundError{}) {
-			// in dualStackOverlay mode, add net routes to windows node
-			if nwInfo.IPV6Mode == string(util.DualStackOverlay) {
+			// add net routes to windows node in dualStackOverlay mode
```

Comment:
nit: update the comment. we don't want to tie this to dualstackoverlay mode. We set these v6 routes if we have v6 enabled.